content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation][2] is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries][3] from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
+
+## Prerequisites
+
+Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
+
+<div class="alert alert-warning">
+When replacing an existing Datadog instrumentation with its OpenTelemetry equivalent, remember to disable the
+Datadog instrumentation to avoid duplicate spans in the trace.
+</div>
+
+<div class="alert alert-info">
+Setting `DD_TRACE_OTEL_ENABLED=true` is not required when using the Datadog SDK for Go.
+</div>
+
+## Language support
+
+| Language | Minimum version required |
+|----------|--------------------------|
+| Java     | v1.35.0                  |
+| Python   | v2.10.0                  |
+| Ruby     | v2.1.0                   |
+| Go       | v1.65.0                  |
+| Node.js  | -                        |
+| PHP      | -                        |
```

Comment:
For PHP it `DD_TRACE_OTEL_ENABLED ` was added in [0.94.0](https://github.com/DataDog/dd-trace-php/releases/tag/0.94.0), lets please use that version to satisfy products expectations.